clisops/ops/subset.py

```diff
@@ -50,47 +48,65 @@ def _subset(ds, args):
 
 
 def subset(
-    ds: Union[xr.Dataset, str, Path],
+    ds: Union[xr.Dataset, str],
```

Comment:
Just want to make a note here that removing `Path` objects here will raise IDE warnings for users using Pathlib. The handling for them is covered on the new line 113.

Reply:
Thanks for pointing this out. I had thought that `Path` objects would raise an error from `utils.map_params` but as you said this is resolved on line 113, before `utils.map_params` is called. I'll put this back in